ca-bundle.crt.metadata

```diff
@@ -109,4 +109,11 @@
 	"key_algo": "ECDSA256",
 	"keystore": "trusted_roots/kitkat.pem"
 }
+{
+	"name":"Ubuntu",
+	"weight": 1,
+	"latest_hash_algo_support": "SHA2",
```

Comment:
sorry I confuse you what I mean is the key name originally should be named as that. But as right now we use "hash_algo" and "key_algo"